Cargo.toml

```diff
@@ -43,6 +43,7 @@ color_quant = "1.1"
 exr = { version = "1.5.0", optional = true }
 qoi = { version = "0.4", optional = true }
 libwebp = { package = "webp", version = "0.2.2", default-features = false, optional = true }
+rayon = { version = "1.8.0", optional = true }
```

Comment:
Are you able to depend on 1.7.0 instead? We currently target a lower MSRV than the 1.8.0 series.

To get the CI to pass, you probably will also have to run something like:
```bash
mv Cargo.lock.msrv Cargo.lock
cargo update -p rayon --precise 1.7.0
mv Cargo.lock Cargo.lock.msrv
```